src/legacy/ui/public/agg_types/controls/order_agg.tsx

```diff
@@ -53,7 +54,7 @@ function OrderAggParamEditor({
 
   return (
     <DefaultEditorAggParams
-      agg={value}
+      agg={value as AggConfig}
```

Comment:
nit: It feels like this casting shouldn't be needed -- did this pop up because TS no longer knew what `paramDef` was?

Reply:
`Type 'AggConfig | undefined' is not assignable to type 'AggConfig'.   Type 'undefined' is not assignable to type 'AggConfig'`

Reply:
i didn't want to go changing any of the editor components so i just used `as`